src/NUnitConsole/nunit4-console/TestEventHandler.cs

```diff
@@ -12,7 +12,7 @@ namespace NUnit.ConsoleRunner
     /// test for the console runner.
     /// </summary>
 #if NETFRAMEWORK
-    public class TestEventHandler : MarshalByRefObject, ITestEventListener
+    public class TestEventHandler : ITestEventListener
 #else
     public class TestEventHandler : ITestEventListener
 #endif
```

Comment:
#if no longer needed as both cases are now the same.

Reply:
Right